jmix-gridexport/gridexport-flowui/src/main/java/io/jmix/gridexportflowui/exporter/entitiesloader/AbstractAllEntitiesLoader.java

```diff
@@ -0,0 +1,137 @@
+/*
+ * Copyright 2023 Haulmont.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.jmix.gridexportflowui.exporter.entitiesloader;
+
+import io.jmix.core.*;
+import io.jmix.core.common.util.Preconditions;
+import io.jmix.core.entity.KeyValueEntity;
+import io.jmix.flowui.data.ContainerDataUnit;
+import io.jmix.flowui.data.DataUnit;
+import io.jmix.flowui.model.*;
+import io.jmix.gridexportflowui.GridExportProperties;
+import io.jmix.gridexportflowui.exporter.EntityExportContext;
+import io.jmix.gridexportflowui.exporter.EntityExporter;
+import org.springframework.transaction.PlatformTransactionManager;
+import org.springframework.transaction.support.TransactionTemplate;
+
+import java.util.List;
+
+/**
+ * Base class for the entity loader which is used to export to other data formats such as excel or json
+ */
+public abstract class AbstractAllEntitiesLoader {
```

Comment:
Why not to make this abstract class implement the `AllEntitiesLoader` interface?